.github/workflows/auto_cherrypick.yml

```diff
@@ -0,0 +1,41 @@
+name: Cherrypick merged pull request
+on:
+  pull_request_target:
+    branches: [master]
+    types: [closed]
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+jobs:
+  cherrypick:
+    name: Cherrypick pull request
+    runs-on: ubuntu-latest
+    # Only run when pull request is merged
+    # or when a comment starting with `/cherrypick` is created by someone other than the
+    # https://github.com/backport-action bot user (user id: 97796249). Note that if you use your
+    # own PAT as `github_token`, that you should replace this id with yours.
+    if: >
+      (
+        github.event_name == 'pull_request_target' &&
+        github.event.pull_request.merged &&
+        github.event.pull_request.milestone != null
+      ) || (
+        github.event_name == 'issue_comment' &&
+        github.event.issue.pull_request &&
+        github.event.comment.user.id != 97796249 &&
+        startsWith(github.event.comment.body, '/backport')
+      )
+    steps:
+      - name: Check if RC branch exists for the milestone
+        id: check_branch
+        run: |
+          BRANCH_NAME="RC${{ github.event.pull_request.number }}.0_master"
+          RESPONSE=$(curl -s -o /dev/null -w "%{http_code}" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${{ github.repository }}/branches/$BRANCH_NAME)
+          if [ $RESPONSE -eq 200 ]; then
+            echo "Branch exists"
+            echo "::set-output name=branch_name::$BRANCH_NAME"
```

Comment:
`set-output` command is deprecated, use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead. https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/